doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3108,15 +3108,15 @@ A curl example using an ``ID``
   export ID=24
 
   curl -H "X-Dataverse-key:$API_TOKEN" -X POST \
-    -F 'jsonData={"description":"My description bbb.","provFreeform":"Test prov freeform","categories":["Data"],"restrict":false}' \
+    -F 'jsonData={"description":"My description bbb.","provFreeform":"Test prov freeform","categories":["Data"],"restrict":false},"dataFileTags":["Survey"]}' \
```

Comment:
This does not parse as valid JSON. Please try it (and the others) at https://jsonlint.com (or your favorite validator):

![Screenshot 2023-10-19 at 2 35 26 PM](https://github.com/IQSS/dataverse/assets/21006/b4211ac2-91df-4b71-b844-39ed01f0b831)